ERCS/erc-7770.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: 7770
+title: Fractional Reserve Token
+description: An interface for fractional reserve tokens.
+author: Yaron Velner (@yaronvel)
+discussions-to: https://ethereum-magicians.org/t/erc-7770-fractional-reserve-token/21103
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-09-17
+requires: 20
+---
+
+
+## Abstract
+We propose a new token standard for synthetic assets that are only partially redeemable to their underlying asset, but fully backed by other collateral assets.
+ 
+The standard defines an interface to mint fractional reserve assets, and a standard to reflect economical risk related data to the token holders and lenders.
+
+## Motivation
+The Cambrian explosion of new L1s and L2s gave rise to bridged assets which are synthetic by nature. Indeed, ETH on Arbitrum L2, or WETH on Binance Smart Chain are not fully fungible with their mainnet counterpart. However, these assets are fully backed by their mainnet counterpart and guaranteed to be redeemable to their mainnet underlying asset, albeit with certain time delay.
+
+Fractional reserve tokens can allow an ecosystem (chains, L2s, and other networks of economic activity) to increase its supply by allowing users to mint the asset not only by bridging it to the ecosystem, but also by borrowing it (typically against a collateral).
+
+As an example, consider a fractional reserve token, namely, frDAI, that represents a synthetic DAI.
+Such token will allow users to mint 1 frDAI upon deposit of 1 DAI, or by providing a collateral that worth more than 1 DAI.
+Quick redemption of frDAI to DAI is available as long as there is still some DAI balance in the frDAI token, and otherwise, the price of frDAI may temporarily fluctuate until borrowers repay their debt.
+
+Fractional reserve tokens may delegate minting capabilities for multiple risk curators and lending markets. Hence, a uniform standard for fractional reserve minting is needed.
+Fractional reserve banking does not come without risks, such as insolvency or a bank run.
+This standard does not aim to dictate economic risk management practices, but rather to have a standard on how to reflect the risk to token holders.
+
+## Specification
+The proposed standard has the following requirements:
```

Comment:
Proposals should be written as if they were already Final. Using "proposed" here would be weird once the standard is accepted.